File: Python/Important-Programs/04-sum-of-numbers-in-given-range.py
num1 = int(input("Enter a start number: "))
num2 = int(input("Enter a End number: "))
sum = 0

for i in range(num1, num2+1):
    sum+=i

print("Sum of", num1, "to", num2, "is:", sum)
 

# Using Recursion
def SumOfRange(sum, num1, num2):
    if num1 > num2:
        return sum
    
    return num1 + SumOfRange(sum, num1+1, num2)

sum=0
print(SumOfRange(sum, num1, num2))

File: Java/1-Important-Programs/04-sum-of-numbers-in-given-range.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner getInput = new Scanner(System.in);

        System.out.println("Enter start number:");
        int num1 = getInput.nextInt();

        System.out.println("Enter end number:");
        int num2 = getInput.nextInt();

        int sum = 0;

        for (int i=num1; i<=num2; i++) {
            sum += i;
        }

        System.out.println("\nSum of " + num1 + " to " + num2 + " is: " + sum);
        getInput.close();
    }
}